<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Account:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;New Account:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newAccountEdit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordEdit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;New Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Check:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkPasswordEdit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>230</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>320</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
